packages/core-typings/src/IUser.ts

```diff
@@ -79,27 +101,37 @@ export interface IUserServices {
 		changedAt: Date;
 	};
 	emailCode?: IUserEmailCode;
-	saml?: {
-		inResponseTo?: string;
-		provider?: string;
-		idp?: string;
-		idpSession?: string;
-		nameID?: string;
-	};
-	ldap?: {
-		id: string;
-		idAttribute?: string;
-	};
-	nextcloud?: {
-		accessToken: string;
-		refreshToken: string;
-		serverURL: string;
-	};
-	dolphin?: {
-		NickName?: string;
-	};
 }
 
+type IUserService = keyof IUserServices;
+type IOAuthServices = keyof IOAuthUserServices;
```

Comment:
```suggestion
type IOAuthService = keyof IOAuthUserServices;
```
I believe singular makes more sense here